synapse/axon.py

```diff
@@ -83,6 +85,33 @@
         resp = await self.cell.has(s_common.uhex(sha256))
         return self.sendRestRetn(resp)
 
+reqValidAxonDel = s_config.getJsValidator({
+    'type': 'object',
+    'properties': {
+        'sha256s': {
+            'type': 'array',
+            'items': {'type': 'string', 'pattern': '(?i)^[0-9a-f]{64}$'}
+        },
+    },
+    'additionalProperties': False,
+    'required': ['sha256s'],
+})
+
+class AxonHttpDelV1(s_httpapi.Handler):
+
+    async def post(self):
+
+        if not await self.reqAuthAllowed(('axon', 'del')):
+            return
+
+        body = self.getJsonBody(validator=reqValidAxonDel)
+        if body is None:
+            return
```

Comment:
Maybe we should return a 400 here?

Reply:
for now, i'd like to keep this constant with the other use cases that pass along application failures with a 200.